crates/hyperswitch_interfaces/src/api.rs

```diff
@@ -38,9 +38,7 @@ use masking::Maskable;
 use router_env::metrics::add_attributes;
 use serde_json::json;
 
-#[cfg(feature = "payouts")]
-pub use self::payouts::*;
-pub use self::{payments::*, refunds::*};
+pub use self::{payments::*, payouts::*, refunds::*};
```

Comment:
Feature flag is needed here to avoid warnings on dead code, let's revert this change